framework/components/ecl-breadcrumbs/_breadcrumbs.scss

```diff
@@ -69,6 +67,10 @@ a.breadcrumb__link {
   // the breadcrumb segment width.
   width: 11px;
   color: #fff;
+
+  &::before {
```

Comment:
![breadcrumbs europa component library](https://cloud.githubusercontent.com/assets/1923476/26544589/9192c658-4462-11e7-9133-bd1fcc6fb4ba.png)

Is as visible now, the arrow will have to be adjusted a bit.